tests/js/content-helper/common/utils/date.test.tsx

```diff
@@ -23,3 +32,76 @@ describe( `${ basePath } canProcessDate()`, (): void => {
 		expect( canProcessDate( new Date( invalidDate ) ) ).toBe( false );
 	} );
 } );
+
+/**
+ * Verifies that getDateInUserLang() returns dates formatted according to the
+ * passed options.
+ */
+describe( `${ basePath } getDateInUserLang()`, (): void => {
+	it( 'should format dates according to passed options', (): void => {
+		expect( getDateInUserLang(
+			new Date( '2022-11-11' ),
+			{ month: 'short', day: 'numeric', year: 'numeric' },
+		) ).toBe( 'Nov 11, 2022' );
+
+		expect( getDateInUserLang(
+			new Date( '2022-12-12' ),
```

Comment:
Should we also add a test with date greater than `12` ... just to verify the proper format?

Reply:
You're right. I updates the existing test.